sdk/communication/Azure.Communication.Common/src/CommunicationTokenRefreshOptions.cs

```diff
@@ -39,6 +46,21 @@ public class CommunicationTokenRefreshOptions
             Argument.AssertNotNull(tokenRefresher, nameof(tokenRefresher));
             RefreshProactively = refreshProactively;
             TokenRefresher = tokenRefresher;
+            RefreshOffsetTime = new TimeSpan(0, 0, DefaultExpiringOffsetMinutes, 0);
```

Comment:
For default values like this, I find it better style to initialize the properties in place instead of inside the constructor.

```c#
internal TimeSpan RefreshOffsetTime { get; } = new TimeSpan(0, 0, DefaultExpiringOffsetMinutes, 0);
```

Reply:
agreed, let's leave the constructor as simple as possible @AikoBB